bin/git-ranch

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+echo "Howdy pardner! Looks like you've mistaken a git command. Did you mean \`git branch\`? See \`git --help\`. Yee-haw!
+
+While yer in the ranchin' mood, why don't ya try this recipe for bacon 'n beans?
```

Comment:
This seems to be excluding the preferences of people who don't eat meat. Please consider adding a `--vegan` option.